iron-iconset-svg.html

```diff
@@ -69,6 +69,10 @@
       }
 
     },
+    
+    attached: function() {
+      this.style.display = 'none';
```

Comment:
ATM `<iron-iconset-svg>` doesn't have a shadow root, so I didn't think I should add one. @cdata?